### ISSUE TITLE
pulseaudio stops working after re-login.

### ISSUE THREAD

**Issue by triantares** [Obsidian-StudiosInc/entrance#47]
## Description
When logging out from a DE and logging in again, pulseaudio fails to restart.
On restarting "entrance" with "killall -HUP entrance" pulseaudio functions as it should. 

This is on Elive 3.8.11 amd64 (Buster based) running either E16 or E23.

- Lightdm does not have this issue -

**Comment by wltjr** [comment]
Hi, thanks for the report. I can confirm that upon logout, pulseaudio is still running. I will have to look into that issue.

**Comment by TheTechRobo** [comment]
I have the similar issue. I left my Elive Beta with E24 manually installed box on for a while, normally after that long a time I have a guru meditation, instead I got kicked back to entrance (E24 probably crashed without handling the exception) and I relogged in. I have no audio, PulseAudio volume control is stuck on "establishing connection to PulseAudio". I have not tried restarting entrance yet.

**Comment by TheTechRobo** [comment]
Nope, it's not because Enlightenment crashed - it's because of me just relogging in. Very strange.

**Comment by Thanatermesis** [comment]
Maybe entrance just booted too early and some "systemd" services needed to be loaded previously?

You can try to delay the boot of entrance like in the "timer" answer of this thread: https://unix.stackexchange.com/questions/397853/how-to-set-a-systemd-unit-to-start-after-loading-the-desktop or play with systemd configurations in some ways to verify if this is the issue (reload systemd configurations are needed to take effect)

**Comment by wltjr** [comment]
No, I am pretty sure there is some disconnect between the session and pulse audio, similar to issue #45. I believe, I need to register the session with logind/elogind. I think that is what is affecting both pulse audio and system actions in the other issue.

**Comment by wltjr** [comment]
Please, can you confirm the issue still exists with latest versions of EFL and Entrance source code, otherwise, this will be closed in ~30 days, thank you!
